rust/backend/ebpf/src/sys_sendmsg.rs

```diff
@@ -0,0 +1,67 @@
+// SPDX-FileCopyrightText: 2024 Tom Weisshuhn <tom.weisshuhn@fau.de>
+//
+// SPDX-License-Identifier: MIT
+
+use aya_ebpf::{macros::{tracepoint, map}, maps::{HashMap, RingBuf}, programs::{TracePointContext}, EbpfContext, helpers::gen::bpf_ktime_get_ns};
+use backend_common::{generate_id, SysSendmsgCall};
+
+#[map(name = "SYS_SENDMSG_MAP")]
+pub static SYS_SENDMSG_MAP: RingBuf = RingBuf::with_byte_size(1024, 0);
```

Comment:
For the future: if you want 1024 events in the buffer, instead of an amount of events that add up to 1024 bytes you can use `mem::size_of::<SysSendmsgCall>() as u32 * 1024`